problems/euler/multiplesOf3And5/java/jailarie/README.md

```diff
@@ -0,0 +1,13 @@
+#Multiples of 3 and 5
+
+https://projecteuler.net/problem=1
+
+If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+
+Find the sum of all the multiples of 3 or 5 below 1000.
+
+##Solution
+Loop through each number from 1 to 999. If the number can be divided by 3 or 5 with a remainder of 0 add it to the sum.
+
+##Running the Program
+Open the Java IDE of your choice. (If you do not have a java IDE installed Eclipse can be downloaded or free. Open eulerProblemOne.java in your IDE and run as Java application. Solution will be printed to the console.
```

Comment:
README looks good - the only issue I can find is a tiny spelling error in line 13- "Eclipse can be downloaded _for_ free"

Reply:
That damn f key! Fixed it. Thanks @cjyovich !

Reply:
Claaaaassic F key!